src/main/java/cz/muni/fi/dao/ComponentDao.java

```diff
@@ -0,0 +1,59 @@
+package cz.muni.fi.dao;
+
+import java.util.List;
+
+/**
+ * @author Matus Macko
+ */
+public interface ComponentDao {
+    /**
+     * List all available components in DB
+     *
```

Comment:
why whitespace here?

Reply:
I checked this on oracle.com and they add a blank line between description and other tags. I guess it is some kind of convention - http://www.oracle.com/technetwork/articles/java/index-137868.html